example/fundamentals/out-dir/1-out-files/build.mill

```diff
@@ -243,8 +243,7 @@ out/mill-server
 
 > cat out/mill-build/methodCodeHashSignatures.dest/current/spanningInvalidationTree.json
 {
-  "call scala.runtime.BoxesRunTime.boxToInteger(int)java.lang.Integer": {},
-  "call scala.Predef$#println(java.lang.Object)void": {
+  "call scala.runtime.BoxesRunTime.boxToInteger(int)java.lang.Integer": {
```

Comment:
Just wanted to mention that I'm not familiar enough with the hash signature stuff to really understand why this change is needed. Oddly enough, that needs to be reverted in https://github.com/com-lihaoyi/mill/pull/4155.